mesonbuild/interpreter/kwargs.py

```diff
@@ -363,6 +363,7 @@ class _BuildTarget(_BaseBuildTarget):
     d_module_versions: T.List[T.Union[str, int]]
     d_unittest: bool
     rust_dependency_map: T.Dict[str, str]
+    swift_interoperability_mode: str
```

Comment:
Thanks!

You could make this `Literal['c', 'cpp']`, but I don't know how much that will help internally.